<launch>

	<node pkg = "imu_rpy" type = "imu_rpy_node" name = "imu_rpy_node" output = "screen">
	
    <param name = "imu_topic" 		type = "string"      value = "/handsfree/imu"/>
    <param name = "mag_topic" 		type = "string"      value = "/handsfree/mag"/>
	
	</node>
	
</launch>

